spec-0000/index.md

```diff
@@ -32,8 +32,8 @@ All versions refer to feature releases (i.e., Python 3.8.0, NumPy 1.19.0; not Py
 
 Specifically, we recommend that:
 
-1. Support for Python versions be dropped **3 years** after their initial release.
-2. Support for core package dependencies be dropped **2 years** after their initial release.
+1. Support for Python versions be dropped **3 years** after the next version of Python is released.
```

Comment:
I am a bit confused about this. This seems like one more year of support for the non-LTS recommendation but I may be missing some context ...

For Python (yearly release cadence), "3 years after the next version is released" means "4 years since the initial release".

The [changes in chart.md](https://github.com/scientific-python/specs/pull/389/files#diff-4216c03ca697c0458f16ef08d3b5d832d4fad4f34565f0aef03526ff1cf84e65) seem to agree with this "one more year of Python support".

Reply:
Good point. When talking about this correction of the _logic_, we've discussed only support for core package dependencies like NumPy. Since the time between releases is typically 6 months, we've actually discussed that we could drop support for an old version 18 months after the next version release. Since the release cadence of Python is annual, we could keep the same nominal time by making this _two_ years after the next release and still keep the same nominal schedule.

So one option is to change these numbers to 1.5 and 2. We could make *both* 2 years for simplicity, but indeed, that would change the nominal support time.

Reply:
Removed. See #409.